apps/opik-backend/src/main/resources/liquibase/db-app-analytics/migrations/000008_add_error_info_to_spans_and_traces.sql

```diff
@@ -0,0 +1,11 @@
+--liquibase formatted sql
+--changeset BorisTkachenko:add_model_to_spans
```

Comment:
Please use a more descriptive name for this changeset e.g: add error info to spans and traces.

Reply:
Copypaste mistake, updated.